### DISCUSSION TITLE
HTML Tag in Markdown

### DISCUSSION THREAD

**Discussion by LethalSnake1337** [rodnansol/spring-configuration-property-documenter#95] [Q&A]
Hi i use in my javadocs the `<p>` Tag which is getting included in the Markdown file how can I fix this?

**Comment by nandorholozsnyak** [comment]
Hello @LethalSnake1337 

There is already a ticket for that, but it is not yet in progress, as we are having multiple formats (markdown, asciidoc, html) I have to take a look into these tags and make sure they can be properly formatted.

https://github.com/rodnansol/spring-configuration-property-documenter/issues/32

I can not promise anything, but I will try to prioritize it.
